app/views/travel_requests/_form.html.erb

```diff
@@ -71,8 +71,11 @@
 </grid-container>
   <!--
     Travel Estimate Form -->
+
   <travel-estimate-form
-    :expenses="<%= request_change_set.estimates_json_form %>"
+    <% unless request_change_set.estimates_json_form == '[]' %>
```

Comment:
@sdellis I wonder if there is a way to avoid this branch in the view.  What would happen if we returned nil from estimates_json_form instead of an empty array?

Reply:
@carolyncole I tested a few different ways to do this, but I don't think any will work out in our context. Do you think this logic should move into a helper?